checkstyle/README.adoc

```diff
@@ -0,0 +1,220 @@
+= checkstyle plugin
+
+== Usage
```

Comment:
update to inform about two different executions, the reason why two and their purpose